components/brave_today/browser/html_parsing_unittest.cc

```diff
@@ -76,7 +76,7 @@ std::string GetHTML() {
 
```

Comment:
can we somehow add a test case for non utf8 encoding?

Reply:
I'll get on this tomorrow :smile: